README.md

```diff
@@ -1,58 +1,52 @@
-<h1 align="center">Dat Desktop</h1>
-
-<div align="center">
-  <strong>Peer to peer data sharing app built for humans.</strong>
-</div>
-
-<br />
-
-<div align="center">
-  <!-- Build Status -->
-  <a href="https://travis-ci.org/dat-land/dat-desktop">
-    <img src="https://img.shields.io/travis/dat-land/dat-desktop/master.svg?style=flat-square"
-      alt="Build Status" />
-  </a>
-  <!-- Standard -->
-  <a href="https://standardjs.com">
-    <img src="https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat-square"
-      alt="Standard" />
-  </a>
-</div>
+# Dat Desktop
+
+**Peer to peer data sharing app built for humans.**
+
+[!["Build Status"](https://img.shields.io/travis/dat-land/dat-desktop/master.svg?style=flat-square)](https://travis-ci.org/dat-land/dat-desktop) 
+[!["Standard](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat-square)](https://standardjs.com)
```

Comment:
I am sure there were aesthetic reasons for this to be there. I am kind-of okay with this change but am not sure how design-people feel about it.

Reply:
If someone wants to change this, let them change it. In Visual Code the Markdown syntax check underlines plain HTML, and I don't like underline code ;-)

IMO this is to small thing to spend much time on.

Reply:
Alright, let's leave it at that